README.md

```diff
@@ -19,7 +19,6 @@ Complete documentation can be found for the main workflow an its subworkflows he
 - [Germline Variant Workflow](./docs/GERMLINE_VARIANT_README.md)
     - [CNV Variant Workflow](./docs/GERMLINE_CNV_README.md)
     - [SNV Variant Workflow](./docs/GERMLINE_SNV_README.md)
-    - [SNV Annotation Workflow](./docs/GERMLINE_SNV_ANNOT_README.md)
```

Comment:
Maybe we should add a blurb about how this leverages the annotation workflow repo.